test/widgets/review/write_review__page_test.dart

```diff
@@ -0,0 +1,242 @@
+import 'dart:math';
+
+import 'package:collection/collection.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/account/models/review.dart';
+import 'package:motis_mitfahr_app/account/pages/reviews_page.dart';
+import 'package:motis_mitfahr_app/account/pages/write_review_page.dart';
+import 'package:motis_mitfahr_app/rides/models/ride.dart';
+import 'package:motis_mitfahr_app/util/buttons/loading_button.dart';
+import 'package:motis_mitfahr_app/util/profiles/reviews/custom_rating_bar.dart';
+import 'package:motis_mitfahr_app/util/supabase_manager.dart';
+import 'package:progress_state_button/progress_button.dart';
+
+import '../../util/factories/profile_factory.dart';
+import '../../util/factories/review_factory.dart';
+import '../../util/factories/ride_factory.dart';
+import '../../util/mocks/mock_server.dart';
+import '../../util/mocks/request_processor.dart';
+import '../../util/mocks/request_processor.mocks.dart';
+import '../../util/pump_material.dart';
+
+void main() {
+  late Profile profile;
+  final MockRequestProcessor processor = MockRequestProcessor();
+
+  setUpAll(() async {
+    MockServer.setProcessor(processor);
+  });
+
+  setUp(() async {
+    profile = ProfileFactory().generateFake(id: 1);
```

Comment:
still not sure why we are hardcoding the ids?

Reply:
We talked about this extensively on one of Philipp's PRs... The probability is astronomically low because we generate an id that is 10 digits long or something like that. But if anything, you can maybe use `profile.id + 1` for this.